airflow/models/dag.py

```diff
@@ -1423,7 +1423,7 @@ def clear(
         confirm_prompt=False,
         include_subdags=True,
         include_parentdag=True,
-        dag_run_state: str = State.RUNNING,
+        dag_run_state: str = State.QUEUED,
```

Comment:
Similarly, we should update L1552 too:

https://github.com/apache/airflow/blob/0ed4445635a70645e53fdf433356829b504e7d53/airflow/models/dag.py#L1552